Sources/ContainerCommands/BuildCommand.swift

```diff
@@ -207,7 +207,33 @@ extension Application {
                     buildFilePath = resolvedPath
                 }
 
-                let buildFileData = try Data(contentsOf: URL(filePath: buildFilePath))
+                let buildFileData: Data
+                let tempFile = FileManager.default.temporaryDirectory.appendingPathComponent("TempDockerfile")
```

Comment:
FYI I finally got back to your #734 PR which is similar!

As with that one, do something like:

```swift
let tempFile = FileManager.default.temporaryDirectory.appendingPathComponent("Dockerfile-\(UUID().uuidString)")
```